erpnext/patches/v14_0/set_sales_person_details_in_opportunity.py

```diff
@@ -0,0 +1,14 @@
+import frappe
+
+def execute():
+    frappe.reload_doc("crm", "doctype", "opportunity")
+
+    frappe.db.sql("""
+        UPDATE `tabOpportunity` AS opp
+        INNER JOIN `tabSales Person` AS sp 
+            ON sp.name = opp.sales_person
+        INNER JOIN `tabEmployee` AS emp 
+            ON emp.name = sp.employee
+        SET opp.sales_person_contact_no = emp.cell_number, 
+            opp.sales_person_email = emp.personal_email
```

Comment:
Pick either `company_email` or `personal_email` .. Give precedence to `company_email`